src/api/java/cvc5/Utils.java

```diff
@@ -151,9 +151,27 @@ public static void validateUnsigned(long[] integers, String name) throws CVC5Api
       String[] pair = rational.split("/");
       return new Pair<>(new BigInteger(pair[0]), new BigInteger(pair[1]));
     }
+    if (rational.contains("."))
+    {
+      String[] pair = rational.split("\\.");
+      if ((new BigInteger(pair[1])).equals(BigInteger.ZERO))
+      {
+        return new Pair<>(new BigInteger(pair[0]), new BigInteger("1"));
+      }
+      BigInteger denominator = BigInteger.TEN.pow(pair[1].length());
+      return new Pair<>(new BigInteger(rational.replace(".", "")), denominator);
+    }
```

Comment:
Is this necessary? I don't think we ever return a "real" value in decimal, but always use a fractional representation.

Reply:
Yes, I encountered a case where there is a decimal point like "1.0"

Reply:
Ah, I've just seen why this happens: https://github.com/cvc5/cvc5/blob/master/src/api/cpp/cvc5.cpp#L3012

Reply:
Ok. By the way, I prefer to have a pair of strings <"1", "2"> than a single string  "1/2". 
I guess most languages does not handle "1/2" properly, so we have to do splitting to convert it to a pair of integers. 
Alternative is to use `std::pair<int64_t, uint64_t>` with the risk of precision loss.

Reply:
```suggestion
```